README-template.md

```diff
@@ -711,7 +715,7 @@ export PROJECT=my-gcp-project
 export BIGLAKE_LOCATION=us
 export BIGLAKE_REGION=us-central1
 export BIGLAKE_CONNECTION=hive-integration-tests
-export BIGLAKE_BUCKET=${USER}-biglake-test
+export BIGLAKE_BUCKET=${PROJECT}-biglake-tests
```

Comment:
Many projects have a `:` in their names, e.g., `google.com:project`, which makes this default bucket name invalid. At least, consider automatically converting `:` to `_` or `-` here.